tkg/client/cluster.go

```diff
@@ -160,7 +160,13 @@ func (c *TkgClient) CreateCluster(options *CreateClusterOptions, waitForCluster
 			return false, errors.Wrap(err, "unable to get cluster configuration")
 		}
 
-		if !c.IsFeatureActivated(constants.FeatureFlagAllowLegacyCluster) {
+		allowLegacyCluster, err := c.TKGConfigReaderWriter().Get(constants.ConfigVariableAllowLegacyCluster)
```

Comment:
Will this get api make sure allow_legacy_cluster set through either feature gate or config file ready to get?

Reply:
@wjun 
If users don't enable allow_legacy_cluster, then our code will set allow_legacy_cluster according to feature gate's value. By default, it's "false".